src/components/LinksSection.jsx

```diff
@@ -0,0 +1,40 @@
+import PropTypes from "prop-types";
+
+const LinkButton = ({ text, link }) => (
+  <a
+    className={`flex items-center justify-center py-2 px-5 rounded-lg font-heading font-normal text-center border-4 border-orange-600 shadow-md hover:bg-orange-600 hover:border-orange-600 w-full text-nowrap`}
+    href={link}
+    rel="noreferrer noopener"
+    target="_blank"
+  >
+    {text}
+  </a>
+);
+
+LinkButton.propTypes = {
+  text: PropTypes.string,
+  link: PropTypes.string,
+};
+
+const LinksSection = ({ className, linksList }) => {
+  return (
+    <div className={className}>
+      <h3 className="text-2xl uppercase font-bold font-heading pb-10">Links</h3>
+
```

Comment:
nit: remove newline, let's try to reserve newlines only at significant breakpoints, e.g. between sections, to make navigation easier

Reply:
personally i think h3 tag and the div tag is 2 different sections, because they refer to a specific part in the component: heading and body respectively. hence why i think the newline should be there

Reply:
by sections, I'm referring to page sections here, i.e. the different components, similar to how the newlines are inserted between page sections in the main site. i did this as a consequence of how I navigate my files in my editor. if you think it makes sense to leave it here though then feel free to do so